api-lib/Gql.ts

```diff
@@ -1,5 +1,6 @@
 import { Gql } from '../src/lib/gql/GqlHasuraAdmin';
-import { NODE_HASURA_ADMIN_SECRET, NODE_HASURA_URL } from './config';
+
+import { LOCAL_HASURA_ADMIN_SECRET, LOCAL_HASURA_URL } from './config';
```

Comment:
This is confusing, and I don't know what the best thing would be, because these api-lib is used by both the serverless functions, therefore they need the `NODE_HASURA_URL` prefix.

But when used for the local scripts, they should be set to the local versions....

I am going to make a fix PR for this

Reply:
I looked at the usages but it was only used by the seed scripts in main, and wasn't sure how this is all mapped out

Reply:
I didn't mean to say that what you did is confusing,
I mean what I implemented and the last minute change lawrence and I made led to a situation that was confusing

I appreciate you working on everything and helping the team get it to a sensible place